project/make/dynbinary

```diff
@@ -4,17 +4,18 @@ set -e
 DEST=$1
 
 if [ -z "$DOCKER_CLIENTONLY" ]; then
+        export IAMSTATIC="false"
```

Comment:
Also, doing the `export` here will have unintended consequences on other builds.  ie, `project/make.sh dynbinary binary` will set `IAMSTATIC=false` and then it'll stay false for building what really is a static binary.  This is why we've got a subshell below.